cmd/kubeadm/app/phases/certs/renewal/certsapi.go

```diff
@@ -130,12 +118,50 @@ func (r *CertsAPIRenewal) Renew(cfg *certutil.Config) (*x509.Certificate, *rsa.P
 		return nil, nil, errors.Errorf("unexpected certificate status: %v", status)
 	}
 
-	cert, err := x509.ParseCertificate(req.Status.Certificate)
+	cert, err := certutil.ParseCertsPEM(req.Status.Certificate)
 	if err != nil {
 		return nil, nil, errors.Wrap(err, "couldn't parse issued certificate")
 	}
 
-	return cert, key, nil
+	if len(cert) != 1 {
+		return nil, nil, errors.Errorf("certificate request %q has %d certificates, wanted exactly 1", req.Name, len(cert))
+	}
+
+	return cert[0], key, nil
+}
+
+func waitForReady(name string, csr certstype.CertificateSigningRequestInterface) error {
```

Comment:
seems weird that this isn't also a cert util, it seems weird that this behavior would on need to live at this location. 

could you add a //TODO to move this to the canonical home for CSRs in 1.14.